File: CS/BlazorServerApp/Data/WeatherForecast.cs
using System;

namespace BlazorServerApp.Data
{
    public class WeatherForecast {
        public DateTime Date { get; set; }

        public int TemperatureC { get; set; }

        public bool Precipitates { get; set; }

        public string Summary { get; set; }

        public double TemperatureF => Math.Round((TemperatureC * 1.8 + 32), 2);
        public string WeatherType { get; set; }
    }
}

File: VB/BlazorServerApp/Data/WeatherForecast.vb
Imports System

Namespace BlazorServerApp.Data
	Public Class WeatherForecast
		Public Property [Date]() As DateTime

		Public Property TemperatureC() As Integer

		Public Property Precipitates() As Boolean

		Public Property Summary() As String

		Public ReadOnly Property TemperatureF() As Double
			Get
				Return Math.Round((TemperatureC * 1.8 + 32), 2)
			End Get
		End Property
		Public Property WeatherType() As String
	End Class
End Namespace
